backend/src/Book/Infrastructure/JsonSchemaValidator.php

```diff
@@ -30,9 +30,11 @@ public function requestBookJsonSchema(): array
             'properties' => [
                 'title' => [
                     'type' => 'string',
+                    'minLength' => 1,
                 ],
                 'author' => [
                     'type' => 'string',
+                    'minLength' => 1,
```

Comment:
**nice-to-have:** bene che avete messo il `minLength` ma ricordate che siamo su due flussi di dati ben distinti.

mai incrociare i flussi 👻

Reply:
![Ghostbuster](http://alinken.people.ua.edu/uploads/8/7/9/2/87929690/published/ghostbusters.jpg?1501582414)